tests/publiccloud/azure_cli.pm

```diff
@@ -54,11 +54,7 @@ sub run {
     # VM creation
     my $vm_create = "az vm create --resource-group $resource_group --name $machine_name --public-ip-sku Standard --tags '$tags'";
     $vm_create .= " --image $image_name --size Standard_B1ms --admin-username azureuser --ssh-key-values ~/.ssh/id_rsa.pub";
-    my $output = script_output($vm_create, timeout => 600, proceed_on_failure => 1);
-    if ($output =~ /ValidationError.*object has no attribute/) {
-        record_soft_failure('bsc#1191482 - Failed to start/stop vms with azure cli');
-        return;
-    }
+    assert_script_run($vm_create, timeout => 600);
```

Comment:
Sorry, but I would have even a better idea: Why not simply remove the `proceed_on_failure => 1`? Then the next check can also remain intact.

```suggestion
    my $output = script_output($vm_create, timeout => 600);
```